.github/workflows/build.yml

```diff
@@ -8,12 +8,16 @@ permissions:
 jobs:
   linux:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python_version: [3.9.x, 3.11.x]
```

Comment:
What does this do? Also it looks good to me, just wanted to learn.

Reply:
This is how one sets up for testing multiple versions of something.

Taken from https://docs.github.com/en/actions/use-cases-and-examples/building-and-testing/building-and-testing-python#using-multiple-python-versions + how we do it in Assimulo.